src/main/java/org/openstreetmap/atlas/checks/validation/relations/OneMemberRelationCheck.java

```diff
@@ -0,0 +1,66 @@
+package org.openstreetmap.atlas.checks.validation.relations;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.openstreetmap.atlas.checks.base.BaseCheck;
+import org.openstreetmap.atlas.checks.flag.CheckFlag;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasObject;
+import org.openstreetmap.atlas.geography.atlas.items.Relation;
+import org.openstreetmap.atlas.geography.atlas.items.RelationMember;
+import org.openstreetmap.atlas.geography.atlas.items.RelationMemberList;
+import org.openstreetmap.atlas.tags.RelationTypeTag;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+
+/**
+ * This check flags {@link Relation}s which contain only one member.
+ *
+ * @author savannahostrowski
+ */
+public class OneMemberRelationCheck extends BaseCheck
+{
+    public static final String OMR_INSTRUCTIONS = "This relation, {0,number,#}, contains only "
+            + "one member.";
+
+    private static final List<String> FALLBACK_INSTRUCTIONS = Collections
+            .singletonList(OMR_INSTRUCTIONS);
+
+    @Override
+    protected List<String> getFallbackInstructions()
+    {
+        return FALLBACK_INSTRUCTIONS;
+    }
+
+    public OneMemberRelationCheck(final Configuration configuration)
+    {
+        super(configuration);
+    }
+
+    @Override
+    public boolean validCheckForObject(final AtlasObject object)
+    {
+        return object instanceof Relation;
+    }
+
+    @Override
+    protected Optional<CheckFlag> flag(final AtlasObject object)
+    {
+        final Relation relation = (Relation) object;
+        final RelationMemberList members = relation.members();
+
+        // If the number of members in the relation is 1, and the relation is either not a
+        // Multipolygon relation or the sole member is role:inner
+        if (members.size() == 1 && (!relation.isMultiPolygon() || members.iterator().next()
```

Comment:
So while this is slick, I would say that that even when the relation is a multi-polygon with one outer member it is worth flagging. While there is nothing about that is prohibited by the OSM spec, a relation is designated to defined relationship between multiple osm elements. Also, a multi-polygon implies multiple polygons. I think the preferred data modeling of this case is just one osm way with all the tags of the relation and no relation.

To make this distinction vs other case it might be worth adding a second flag instruction to cover this.

Reply:
@jklamer I agree, sounds good.